scheduled-jobs-alert.tf

```diff
@@ -133,3 +133,24 @@ module "check-new-envelopes-alert" {
   trigger_threshold          = 0
   resourcegroup_name         = "${azurerm_resource_group.rg.name}"
 }
+
+module "send-notifications-alert" {
+  source            = "git@github.com:hmcts/cnp-module-metric-alert"
+  location          = "${azurerm_application_insights.appinsights.location}"
+  app_insights_name = "${azurerm_application_insights.appinsights.name}"
+
+  enabled    = "${var.env == "prod"}"
+  alert_name = "Send-Notifications"
+  alert_desc = "Triggers when no logs from send-notifications job found within timeframe."
+
+  app_insights_query = "traces | where message startswith 'Started send-notifications job'"
+
+  frequency_in_minutes       = 120
+  time_window_in_minutes     = 120
```

Comment:
tasks is every 10mins unless there's a plan to override in prod. perhaps enough to run alert every 20mins for 30min window. just in case some weird latency is present